api/src/main/java/io/druid/data/input/impl/TimestampSpec.java

```diff
@@ -47,12 +47,12 @@
 
   private final String timestampColumn;
   private final String timestampFormat;
-  private final Function<Object, DateTime> timestampConverter;
   // this value should never be set for production data
   private final DateTime missingValue;
+  private final transient Function<Object, DateTime> timestampConverter;
```

Comment:
It was for indication that this field is not checked in equals() and hashCode(). But that was a bad way, removed `transient` and added a simple comment.